doc/sphinx/addendum/type-classes.rst

```diff
@@ -295,10 +295,18 @@ the Existing Instance command to achieve the same effect.
 Summary of the commands
 -----------------------
 
-.. cmd:: Class @inductive_definition {* with @inductive_definition }
+.. cmd:: Class @record_definition
+         Class @class_definition
 
-   The :cmd:`Class` command is used to declare a typeclass with parameters
-   :n:`{* @binder }` and fields the declared record fields.
+   .. insertprodn class_definition class_definition
+
+   .. prodn::
+      class_definition ::= {? > } @ident_decl {* @binder } {? : @sort } {? := @constructor } {? @decl_notations }
+
+   In the first form, declares a typeclass with parameters
+   :n:`{* @binder }` and the listed record fields.  In the second form ...
+
+   .. todo PR how to finish the sentence above?
```

Comment:
The first form should be described as simply declaring a normal record and making this record a type class. The second one is already described below under `singleton-class` so its description can simply be moved up a bit.